db/seeds/player_seeder.rb

```diff
@@ -0,0 +1,37 @@
+class PlayerSeeder
```

Comment:
Should rename this and wrap it in a rake task so that it can be run ad-hoc or on schedule to update players.